src/main/java/com/smartagilify/core/model/BaseDTO.java

```diff
@@ -1,29 +1,19 @@
 package com.smartagilify.core.model;
 
 import com.smartagilify.core.enumerations.EN_STATE;
-import io.swagger.annotations.ApiModel;
-import io.swagger.annotations.ApiModelProperty;
 import lombok.Data;
 import lombok.NoArgsConstructor;
 import lombok.experimental.SuperBuilder;
 
```

Comment:
validation should be added to this class as well

Reply:
why? Is there a field that should annotate with validation annotations?